opencti-platform/opencti-graphql/src/domain/backgroundTask-common.js

```diff
@@ -33,30 +35,32 @@ const areParentTypesKnowledge = (parentTypes) => parentTypes && parentTypes.flat
 // check a user has the right to create a list or a query background task
 export const checkActionValidity = async (context, user, input, scope, taskType) => {
   const { actions, filters: baseFilterObject, ids } = input;
-  const filters = (baseFilterObject && JSON.parse(baseFilterObject)?.filters) ?? [];
+  const filters = isFilterGroupNotEmpty(baseFilterObject)
+    ? (JSON.parse(baseFilterObject)?.filters ?? [])
+    : [];
```

Comment:
baseFilterObject seems to be a string, no ?
if so, isFilterGroupNotEmpty(baseFilterObject) is not relevant